spec/actors/hyrax/actors/attach_members_actor_spec.rb

```diff
@@ -17,77 +17,79 @@
   end
 
   describe "#update" do
-    before { work.ordered_members << existing_child_work }
+    context 'when using an active_fedora work' do
```

Comment:
i'm not sure why the smart diff rendering is doing so much work here. the diff is:

  - this `context` is added beginning on L20 and wrapping all the tests that aren't already wrapped by the valkyrie context.
  - everything within is indented 2 spaces, and otherwise unchanged.